.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Feature Request
+about: Want us to add any features to Jekyll?
+---
+
+<!--
+  Hi! Thanks for considering to file a feature request with Jekyll. Please take the time to
+  answer the basic questions. Please try to be as detailed as possible.
+
+  Thanks!
+-->
+
+## Summary
+
+<!--
+  A one-paragraph explanation of the feature.
+-->
+
+## Motivation
+
+<!--
+  Why do you want to see this feature in Jekyll? What makes you sure that it should not be
+  implemented at the plugin level, but in Jekyll core? What use cases does it support?
+-->
+
```

Comment:
The majority of the reporters are probably going to answer this with something along the lines of:
```
"Because GitHub Pages doesn't allow an existing plugin to be used.."
```

Reply:
true, let's add a note there